### ISSUE TITLE
Tile - Avoid the use of implicit headings - (2036570305)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7841]
### Violation: 
 Avoid the use of implicit headings 

![image](https://github.com/Esri/calcite-design-system/assets/5023024/c23b2d5b-e277-4b82-879b-b4a1469ca035)

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) 

### Severity: 
 6 

### Media Type: 
 Page Structure 

--- 
### Areas for Remediation: 
- Title: Tile - Avoid the use of implicit headings - (2036570305) 
- Module: Tile 
**Issue**
There is content "River" that functions as a heading but does not use heading markup.

**User Impact**
Screen reader users will have difficulty efficiently navigating and gaining an accurate overview of the page.

**Code Reference** 
 ```
<div class="heading">Rivers</div>
 ``` 
**Suggestion**
Avoid the use of implicit headings. Text that visually appears as a heading for a section of content must use ``` <h1> ```, ``` <h2> ```, etc. elements. Additionally, ensure the level of the heading accurately reflects the element&#039;s position in the visual hierarchy.

**Compliant Code Example** 
 ```
<h4 class="heading">Rivers</h4>
 ```
 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
This has been mitigated prior to the `2.0.0` release: https://codepen.io/geospatialem/pen/ZEPQbrr
